scripts/jenkins/jobs-ibs/openstack-mkcloud.yaml

```diff
@@ -0,0 +1,489 @@
+         choices:
+            - default
+            - none
+            - swift
+            - ceph
+ 
+     - string:
+         name: github_pr
+         description: "String is a ':' separated list of these vaules: $ORG/$repo:$PR_ID:$SHA1:$BRANCH:$context Note: SHA1 must be latest commit in PR, $context is optional  (only use if multiple gating runs needed per PR)"
+    
+     - string:
+         name: UPDATEREPOS
+         default:
+         description: 'Update repositories (one URL per line)'
+
+     - bool:
+         name: UPDATEBEFOREINSTALL
+         default: false
+         description: 'add update repos before crowbar install'
+     
+     - string:
+         name: TESTHEAD
+         default : '1'
+         description: 'if non-empty, test latest version from Devel:Cloud (:1.0)'
+   
+     - string:
+         name: nodenumber
+         default: '2'
+         description: number of nodes excl. admin-node minimum 2=controller+compute
+
+     - string:
+         name: nodenumbercompute
+         default:
+         description: Number of compute  nodes
+
+     - string: 
+         name: want_tempes
+         default: 
+         description: can be used for force off(0) or on(1)
+
+     - string: 
+         name: tempestoptions
+         default:
+         description: |
+           Additional options to pass to the \"run_tempest.sh\
+           \ -N \" step.\nUse just \"-t\" for a full tempest run.\nDefault: -t -s.
+
+     - string: 
+         name: ostestroptions
+         default: -t -s
+         description: |
+            If given, a extra ostestr run is executed (after the default tempest run).
+            Use i.e. " --regex '^magnum.tests.functional.api'"
+
+     - string: 
+         name: want_horizon_integration_test
+         default:
+         description: |
+             Execute selenium integration tests for Horizon after Tempest. This test is in beta, and
+             a fail will not be considered for now.
+  
+     - string: 
+         name: want_sles12
+         default:
+         description: Set to 1 if sle12 nodes should be installed
+
+     - string: 
+         name: want_sles12sp2
+         default:
+         description: deploy cloud7 with SP2 instead of SP1
+
+     - string: 
+         name: upgrade_cloudsource
+         default:
+         description: In case of using upgrade, upgrade to that cloudsource version
+
+     - string: 
+         name: hacloud
+         default:
+         description: If hacloud=1 then deploy a 4-node cluster
+
+     - bool: 
+         name: WITHCROWBARREGISTER
+         default: false
+         description: boot up one lonely node in the admin network and register it with crowbar_register (image type depends on want_sles12 parameter)
+
+     - string: 
+         name: want_all_ssl
+         default:
+         description: set to 1 to deploy all cloud services with SSL aka HTTPS
+
+     - choice: 
+         name: cinder_backend
+         choices:
+           - local
```

Comment:
the original one have also the empty line as a valid choice, nos sure if needs to be added here